src/main/java/org/folio/edge/controller/exception/ExceptionHandlerController.java

```diff
@@ -52,7 +54,7 @@ public Error handleMissingHeaderException(MissingRequestHeaderException e) {
 
   @ExceptionHandler(BadCredentialsException.class)
   @ResponseStatus(HttpStatus.UNAUTHORIZED)
-  public Error handleBadCredentialsException(BadCredentialsException e){
+  public Error handleBadCredentialsException(BadCredentialsException e) {
```

Comment:
I think we should return api errors in format expected by central server
. You can check the schema here https://raw.githubusercontent.com/folio-org/mod-inn-reach/master/src/main/resources/swagger.api/schemas/d2ir/innReachResponseDTO.json

Reply:
I checked documentation, it seems we have two types of error responces.
The first one implemented now is for authentication errors.
The second one you listed is for common mistakes.
I think we need to implement the second option as well, as well as refactor the code to use the appropriate error response for different cases. I would create a separate task for this.